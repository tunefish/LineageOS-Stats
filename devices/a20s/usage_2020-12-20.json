{
  "17.1-20200715-unofficial-tissot": 1501,
  "17.1-20200821-unofficial-z3": 922
}