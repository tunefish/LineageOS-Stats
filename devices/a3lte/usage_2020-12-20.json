{
  "14.1-20171111-nightly-a3lte": 3,
  "15.1-20180518-unofficial-a33g": 110,
  "15.1-20180518-unofficial-a3lte": 134,
  "15.1-20180518-unofficial-a3ulte": 242,
  "15.1-20201128-unofficial-herolte": 1,
  "16.0-20181122-unofficial-a3lte": 145,
  "17.1-20200715-unofficial-tissot": 6,
  "17.1-20200716-unofficial-herolte": 4,
  "17.1-20200720-unofficial-a3lte": 48,
  "17.1-20200819-unofficial-a3lte": 384,
  "17.1-20200821-unofficial-z3": 19,
  "17.1-20201112-unofficial-ef63": 1,
  "17.1-20201116-unofficial-z3": 1,
  "17.1-20201122-unofficial-herolte": 1,
  "18.1-20200819-official-a3lte": 1
}