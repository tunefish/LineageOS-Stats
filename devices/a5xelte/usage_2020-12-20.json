{
  "0.13-q-2020102781252-test-a5xelte": 1,
  "15.1-20190426-unofficial-a5xelte": 1,
  "15.1-20190517-unofficial-a5xelte": 3,
  "15.1-20190526-unofficial-a5xelte": 4,
  "15.1-20190623-unofficial-a5xelte": 3,
  "15.1-20190709-unofficial-a5xelte": 14,
  "15.1-20201128-unofficial-herolte": 3,
  "16.0-20190121-unofficial-a5xelte": 15,
  "16.0-20190625-unofficial-a5xelte": 10,
  "16.0-20190627-unofficial-a5xelte": 3,
  "16.0-20190707-unofficial-a5xelte": 8,
  "16.0-20190723-unofficial-a5xelte": 6,
  "16.0-20190811-unofficial-a5xelte": 2,
  "16.0-20190816-unofficial-a5xelte": 2,
  "16.0-20190818-unofficial-a5xelte": 40,
  "16.0-20190824-nightly-a5xelte": 1,
  "16.0-20190827-nightly-a5xelte": 1,
  "16.0-20190903-nightly-a5xelte": 1,
  "16.0-20190904-nightly-a5xelte": 2,
  "16.0-20190905-nightly-a5xelte": 1,
  "16.0-20190908-nightly-a5xelte": 2,
  "16.0-20190909-unofficial-a5xelte": 1,
  "16.0-20190913-nightly-a5xelte": 5,
  "16.0-20190914-nightly-a5xelte": 1,
  "16.0-20190916-nightly-a5xelte": 1,
  "16.0-20190917-nightly-a5xelte": 2,
  "16.0-20190921-nightly-a5xelte": 2,
  "16.0-20190923-nightly-a5xelte": 1,
  "16.0-20190925-nightly-a5xelte": 1,
  "16.0-20190926-nightly-a5xelte": 1,
  "16.0-20190930-nightly-a5xelte": 1,
  "16.0-20191002-nightly-a5xelte": 1,
  "16.0-20191009-nightly-a5xelte": 1,
  "16.0-20191013-nightly-a5xelte": 2,
  "16.0-20191015-nightly-a5xelte": 2,
  "16.0-20191016-nightly-a5xelte": 1,
  "16.0-20191018-nightly-a5xelte": 1,
  "16.0-20191022-nightly-a5xelte": 3,
  "16.0-20191026-nightly-a5xelte": 1,
  "16.0-20191029-nightly-a5xelte": 3,
  "16.0-20191030-nightly-a5xelte": 1,
  "16.0-20191101-nightly-a5xelte": 2,
  "16.0-20191104-nightly-a5xelte": 1,
  "16.0-20191105-nightly-a5xelte": 4,
  "16.0-20191107-nightly-a5xelte": 1,
  "16.0-20191109-nightly-a5xelte": 1,
  "16.0-20191110-nightly-a5xelte": 4,
  "16.0-20191112-nightly-a5xelte": 3,
  "16.0-20191114-nightly-a5xelte": 1,
  "16.0-20191116-nightly-a5xelte": 1,
  "16.0-20191121-nightly-a5xelte": 2,
  "16.0-20191122-nightly-a5xelte": 4,
  "16.0-20191126-nightly-a5xelte": 1,
  "16.0-20191127-nightly-a5xelte": 1,
  "16.0-20191130-nightly-a5xelte": 3,
  "16.0-20191201-nightly-a5xelte": 1,
  "16.0-20191205-nightly-a5xelte": 8,
  "16.0-20191206-nightly-a5xelte": 1,
  "16.0-20191207-nightly-a5xelte": 4,
  "16.0-20191209-nightly-a5xelte": 2,
  "16.0-20191210-nightly-a5xelte": 4,
  "16.0-20191211-nightly-a5xelte": 2,
  "16.0-20191214-nightly-a5xelte": 2,
  "16.0-20191215-nightly-a5xelte": 2,
  "16.0-20191219-nightly-a5xelte": 1,
  "16.0-20191222-nightly-a5xelte": 3,
  "16.0-20191224-nightly-a5xelte": 4,
  "16.0-20191227-nightly-a5xelte": 3,
  "16.0-20191228-nightly-a5xelte": 2,
  "16.0-20191229-nightly-a5xelte": 2,
  "16.0-20191230-nightly-a5xelte": 1,
  "16.0-20191231-nightly-a5xelte": 3,
  "16.0-20200101-nightly-a5xelte": 1,
  "16.0-20200103-nightly-a5xelte": 2,
  "16.0-20200106-nightly-a5xelte": 3,
  "16.0-20200107-nightly-a5xelte": 1,
  "16.0-20200109-nightly-a5xelte": 1,
  "16.0-20200111-nightly-a5xelte": 2,
  "16.0-20200113-nightly-a5xelte": 2,
  "16.0-20200115-nightly-a5xelte": 2,
  "16.0-20200116-nightly-a5xelte": 2,
  "16.0-20200117-nightly-a5xelte": 1,
  "16.0-20200118-nightly-a5xelte": 5,
  "16.0-20200119-nightly-a5xelte": 1,
  "16.0-20200121-nightly-a5xelte": 1,
  "16.0-20200122-nightly-a5xelte": 1,
  "16.0-20200123-nightly-a5xelte": 1,
  "16.0-20200124-nightly-a5xelte": 3,
  "16.0-20200130-nightly-a5xelte": 1,
  "16.0-20200203-microg-a5xelte": 2,
  "16.0-20200203-nightly-a5xelte": 1,
  "16.0-20200204-nightly-a5xelte": 2,
  "16.0-20200205-nightly-a5xelte": 2,
  "16.0-20200207-nightly-a5xelte": 2,
  "16.0-20200208-nightly-a5xelte": 2,
  "16.0-20200209-nightly-a5xelte": 1,
  "16.0-20200212-nightly-a5xelte": 1,
  "16.0-20200213-nightly-a5xelte": 1,
  "16.0-20200215-nightly-a5xelte": 1,
  "16.0-20200216-nightly-a5xelte": 1,
  "16.0-20200217-nightly-a5xelte": 1,
  "16.0-20200218-nightly-a5xelte": 1,
  "16.0-20200220-nightly-a5xelte": 2,
  "16.0-20200221-nightly-a5xelte": 3,
  "16.0-20200222-nightly-a5xelte": 3,
  "16.0-20200224-nightly-a5xelte": 9,
  "16.0-20200229-nightly-a5xelte": 2,
  "16.0-20200301-nightly-a5xelte": 2,
  "16.0-20200303-nightly-a5xelte": 1,
  "16.0-20200306-nightly-a5xelte": 1,
  "16.0-20200307-nightly-a5xelte": 2,
  "16.0-20200308-nightly-a5xelte": 2,
  "16.0-20200309-nightly-a5xelte": 3,
  "16.0-20200310-nightly-a5xelte": 3,
  "16.0-20200311-nightly-a5xelte": 7,
  "16.0-20200312-nightly-a5xelte": 1,
  "16.0-20200313-nightly-a5xelte": 8,
  "16.0-20200314-nightly-a5xelte": 2,
  "16.0-20200315-nightly-a5xelte": 1,
  "16.0-20200317-nightly-a5xelte": 5,
  "16.0-20200319-nightly-a5xelte": 2,
  "16.0-20200320-nightly-a5xelte": 2,
  "16.0-20200321-nightly-a5xelte": 2,
  "16.0-20200322-nightly-a5xelte": 2,
  "16.0-20200323-microg-a5xelte": 1,
  "16.0-20200324-nightly-a5xelte": 7,
  "16.0-20200325-nightly-a5xelte": 33,
  "16.0-20200331-microg-a5xelte": 1,
  "16.0-20200412-nightly-a5xelte": 12,
  "16.0-20200419-nightly-a5xelte": 194,
  "16.0-20200422-microg-a5xelte": 1,
  "16.0-20200427-nightly-a5xelte": 20,
  "16.0-20200502-microg-a5xelte": 1,
  "16.0-20200507-microg-a5xelte": 1,
  "16.0-20200521-microg-a5xelte": 2,
  "17.0-20200108-unofficial-a5xelte": 1,
  "17.1-20200117-unofficial-a5xelte": 14,
  "17.1-20200121-unofficial-a5xelte": 8,
  "17.1-20200207-unofficial-a5xelte": 1,
  "17.1-20200210-unofficial-a5xelte": 3,
  "17.1-20200216-unofficial-a5xelte": 1,
  "17.1-20200222-unofficial-a5xelte": 1,
  "17.1-20200229-unofficial-a5xelte": 9,
  "17.1-20200307-unofficial-a5xelte": 2,
  "17.1-20200312-unofficial-a5xelte": 10,
  "17.1-20200412-unofficial-a5xelte": 21,
  "17.1-20200516-unofficial-a5xelte": 176,
  "17.1-20200529-nightly-a5xelte": 1,
  "17.1-20200530-nightly-a5xelte": 4,
  "17.1-20200601-nightly-a5xelte": 8,
  "17.1-20200604-nightly-a5xelte": 7,
  "17.1-20200606-unofficial-a5xelte": 1,
  "17.1-20200607-nightly-a5xelte": 13,
  "17.1-20200614-nightly-a5xelte": 9,
  "17.1-20200621-nightly-a5xelte": 7,
  "17.1-20200628-nightly-a5xelte": 30,
  "17.1-20200712-nightly-a5xelte": 19,
  "17.1-20200715-unofficial-tissot": 16,
  "17.1-20200716-unofficial-herolte": 12,
  "17.1-20200719-nightly-a5xelte": 21,
  "17.1-20200726-nightly-a5xelte": 45,
  "17.1-20200809-nightly-a5xelte": 75,
  "17.1-20200818-microg-a5xelte": 7,
  "17.1-20200821-unofficial-z3": 57,
  "17.1-20200824-microg-a5xelte": 2,
  "17.1-20200830-nightly-a5xelte": 45,
  "17.1-20200904-unofficial-a5xelte": 1,
  "17.1-20200906-nightly-a5xelte": 32,
  "17.1-20200911-unofficial-jasmine_sprout": 2,
  "17.1-20200913-nightly-a5xelte": 72,
  "17.1-20200918-unofficial-a5xelte": 1,
  "17.1-20200920-nightly-a5xelte": 240,
  "17.1-20200921-microg-a5xelte": 12,
  "17.1-20200927-nightly-a5xelte": 219,
  "17.1-20201004-nightly-a5xelte": 170,
  "17.1-20201011-nightly-a5xelte": 190,
  "17.1-20201012-microg-a5xelte": 11,
  "17.1-20201014-unofficial-a5xelte": 2,
  "17.1-20201014-unofficial-herolte": 1,
  "17.1-20201016-unofficial-a5xelte": 1,
  "17.1-20201017-unofficial-a5xelte": 2,
  "17.1-20201018-nightly-a5xelte": 146,
  "17.1-20201023-unofficial-a5xelte": 1,
  "17.1-20201025-nightly-a5xelte": 182,
  "17.1-20201101-nightly-a5xelte": 174,
  "17.1-20201105-microg-a5xelte": 6,
  "17.1-20201108-nightly-a5xelte": 143,
  "17.1-20201112-unofficial-ef63": 1,
  "17.1-20201113-unofficial-tissot": 2,
  "17.1-20201115-nightly-a5xelte": 198,
  "17.1-20201116-unofficial-z3": 5,
  "17.1-20201121-unofficial-a5xelte": 1,
  "17.1-20201122-nightly-a5xelte": 181,
  "17.1-20201122-unofficial-herolte": 4,
  "17.1-20201129-nightly-a5xelte": 210,
  "17.1-20201130-microg-a5xelte": 14,
  "17.1-20201206-nightly-a5xelte": 229,
  "17.1-20201213-nightly-a5xelte": 335,
  "17.1-20201214-unofficial-a5xelte": 1,
  "17.1-20201218-unofficial-a5xelte": 1,
  "17.1-20201219-unofficial-a5xelte": 1,
  "18.0-20200927-unofficial-a5xelte": 1,
  "18.0-20201013-unofficial-a5xelte": 2,
  "18.0-20201022-unofficial-a5xelte": 2,
  "18.0-20201112-unofficial-a5xelte": 1,
  "18.0-20201122-unofficial-a5xelte": 7
}