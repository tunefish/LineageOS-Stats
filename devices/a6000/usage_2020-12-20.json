{
  "14.1-20170125-unofficial-a6000": 3,
  "14.1-20170205-unofficial-a6000": 46,
  "14.1-20170210-unofficial-a6000": 10,
  "14.1-20170225-unofficial-a6000": 35,
  "14.1-20170228-unofficial-a6000": 4,
  "14.1-20170313-unofficial-a6000": 2,
  "14.1-20170403-unofficial-a6000": 26,
  "14.1-20170422-nightly-a6000": 6,
  "14.1-20170514-unofficial-a6000": 3,
  "14.1-20170607-unofficial-a6000": 2,
  "14.1-20170614-unofficial-a6000": 2,
  "14.1-20170705-unofficial-a6000": 4,
  "14.1-20170706-unofficial-a6000": 38,
  "14.1-20170731-unofficial-hyperteam-a6000": 1,
  "14.1-20170813-unofficial-a6000": 1,
  "14.1-20170831-unofficial-a6000": 38,
  "15.1-20180210-hyper-a6000": 5,
  "15.1-20180216-hyper-a6000": 10,
  "15.1-20180308-hyper-a6000": 6,
  "15.1-20180413-hyper-a6000": 18,
  "15.1-20180514-unofficial-a6000": 12,
  "15.1-20180604-hyper-a6000": 1,
  "15.1-20180611-hyper-a6000": 3,
  "15.1-20180720-hyper-a6000": 2,
  "15.1-20180724-hyper-a6000": 8,
  "15.1-20180807-hyper-stable-a6000": 70,
  "15.1-20180807-hyper-stable-rendang": 1,
  "15.1-20200522-unofficial-a6000": 16,
  "16.0-20190302-unofficial-a6000": 3,
  "16.0-20190309-hyper-a6000": 4,
  "16.0-20190316-hyper-a6000": 5,
  "16.0-20190417-unofficial-a6000": 5,
  "16.0-20190509-unofficial-a6000": 6,
  "16.0-20190610-unofficial-a6000": 17,
  "16.0-20190610-unofficial-a6010": 30,
  "16.0-20190808-unofficial": 2,
  "16.0-20190808-unofficial-a6000": 256,
  "17.0-20200102-unofficial-a6000": 4,
  "17.1": 1,
  "17.1-20200121-unofficial-a6000": 92,
  "17.1-20200205-unofficial-a6000": 742,
  "17.1-20200206-unofficial-a6000": 5,
  "17.1-20200907-unofficial-a6000": 4,
  "17.1-20200913-unofficial-a6000": 1,
  "17.1-20200916-unofficial-a6000": 55,
  "17.1-20201024-unofficial-a6000": 19,
  "17.1-20201104-unofficial-a37": 2,
  "17.1-20201112-unofficial-a6000": 1,
  "17.1-20201113-unofficial-a6000": 5,
  "18.0-20201004-unofficial-a6000": 1,
  "18.0-20201018-unofficial-a6000": 8,
  "18.0-20201021-unofficial-a6000": 3,
  "18.0-20201025-unofficial-a6000": 9,
  "18.0-20201121-unofficial-a6000": 28,
  "lineage_a6000-userdebug 10 qq3a.200805.001 eng.xvae.20201118.051554 test-keys": 16,
  "lineage_a6000-userdebug 10 qq3a.200805.001 eng.xvae.20201217.052544 test-keys": 1,
  "lineage_a6000-userdebug 7.1.2 njh47f 972fe43752 test-keys": 3,
  "nuclearom-v-unofficial[7.1.2]--20171117-2103": 7,
  "rr_a6000-userdebug 10 qp1a.191105.004 eng.acrore.20191229.172644 test-keys": 22
}