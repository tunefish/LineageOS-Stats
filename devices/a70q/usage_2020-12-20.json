{
  "16.0-20191013-unofficial-a70q": 1,
  "16.0-20191014-unofficial-a70q": 21,
  "17.0-20191203-unofficial-a70q": 3,
  "17.0-20191230-unofficial-a70q": 1,
  "17.1-20200213-unofficial-a70q": 657,
  "17.1-20200715-unofficial-tissot": 1,
  "17.1-20200717-unofficial-a70q": 9,
  "17.1-20200820-unofficial-a70q": 4,
  "17.1-20200821-unofficial-z3": 16,
  "17.1-20201009-unofficial-a70q": 2,
  "17.1-20201010-unofficial-a70q": 2,
  "17.1-20201013-unofficial-a70q": 5,
  "17.1-20201014-unofficial-a70q": 1,
  "17.1-20201015-unofficial-a70q": 2,
  "17.1-20201020-unofficial-a70q": 1,
  "17.1-20201103-unofficial-a70q": 22,
  "17.1-20201104-unofficial-a70q": 3,
  "17.1-20201106-unofficial-a70q": 5,
  "17.1-20201111-unofficial-a70q": 117,
  "17.1-20201119-unofficial-a70q": 2,
  "17.1-20201120-unofficial-a70q": 2,
  "17.1-20201121-unofficial-a70q": 6,
  "17.1-20201123-unofficial-a70q": 5,
  "17.1-20201124-unofficial-a70q": 5,
  "17.1-20201125-unofficial-a70q": 5,
  "17.1-20201128-unofficial-a70q": 1,
  "17.1-20201206-unofficial-a70q": 110,
  "17.1-20201218-unofficial-a70q": 4,
  "17.1-20201219-unofficial-a70q": 2,
  "treble_arm64_bvn-userdebug 10 qq3a.200805.001 eng.luke_u.20200815.070014 test-keys": 1,
  "treble_arm64_bvn-userdebug 10 qq3a.200905.001 eng.igor.20201013.190840 test-keys": 4,
  "treble_arm64_bvn-userdebug 10 qq3a.200905.001 eng.igor.20201019.185735 test-keys": 4,
  "treble_arm64_bvn-userdebug 10 qq3a.200905.001 eng.igor.20201111.184406 test-keys": 1
}