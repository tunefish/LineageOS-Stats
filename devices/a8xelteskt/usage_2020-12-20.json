{
  "17.1-20200715-unofficial-tissot": 1468,
  "17.1-20200716-unofficial-herolte": 1,
  "17.1-20200821-unofficial-z3": 927,
  "17.1-20201015-unofficial-starlte": 1
}