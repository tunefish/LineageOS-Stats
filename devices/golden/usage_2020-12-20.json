{
  "13.0-20170118-unofficial-golden": 1,
  "13.0-20170121-unofficial-golden": 1,
  "13.0-20170129-unofficial-golden": 89,
  "13.0-20170224-unofficial-golden": 5,
  "13.0-20170302-unofficial-golden": 8,
  "13.0-20170310-unofficial-golden": 21,
  "13.0-20170328-unofficial-golden": 730,
  "13.0-20170420-unofficial-golden": 49,
  "14.1-20170122-unofficial-golden": 256,
  "14.1-20170318-unofficial-golden": 4,
  "14.1-20170402-unofficial-golden": 10,
  "14.1-20170415-unofficial-golden": 4,
  "14.1-20170419-unofficial-golden": 5,
  "14.1-20170503-unofficial-golden": 1,
  "14.1-20170507-unofficial-golden": 16,
  "14.1-20170522-unofficial-golden": 1,
  "14.1-20170609-unofficial-golden": 452,
  "14.1-20170610-unofficial-golden": 7,
  "14.1-20170628-unofficial-golden": 745,
  "14.1-20170724-unofficial-golden": 46,
  "14.1-20170817-unofficial-golden": 30,
  "14.1-20170920-unofficial-golden": 7,
  "14.1-20171011-unofficial-golden": 68,
  "14.1-20171215_192311-official-golden": 220,
  "14.1-20171215_192311-unofficial-golden": 824,
  "14.1-20180109_002536-unofficial-golden": 181,
  "14.1-20181014-unofficial-golden": 3,
  "14.1-20200817_071197-official-golden-rogerwed": 1,
  "15.1-20181030-unofficial-golden": 5,
  "17.1-20201113-unofficial-tissot": 2,
  "golden-user 8.1.0 oe632e 3de271a2ce release-keys": 1,
  "njh47f release-keys": 1
}